controllers/storagecluster/storageconsumer.go

```diff
@@ -44,6 +45,48 @@ func (s *storageConsumer) ensureCreated(r *StorageClusterReconciler, storageClus
 			{Name: util.GenerateNameForGroupSnapshotClass(storageCluster, util.RbdGroupSnapshotter)},
 			{Name: util.GenerateNameForGroupSnapshotClass(storageCluster, util.CephfsGroupSnapshotter)},
 		}
+
+		if r.AvailableCrds[VirtualMachineCrdName] {
```

Comment:
We are moving away from AvailableCrds. Please replace this with a Get for the CRD obj